package.json

```diff
@@ -2,6 +2,7 @@
   "name": "solid-primitives",
   "private": true,
   "description": "A collection of high-quality, community contributed building blocks.",
+  "homepage": "https://primitives.solidjs.community/",
```

Comment:
I understand this is not a package, but still it might be nice to have the homepage here.